api/reqs/models.py

```diff
@@ -69,6 +69,34 @@ class PolicyTypes(Enum):
     strategy = 'Strategy'
 
 
+@unique
+class WorkflowPhases(Enum):
+    """
+    We're assuming two potential workflows.
+
+    Import Workflow
+
+    0.  Policy exists in DB, but no document is associated with it. (blank)
```

Comment:
Did you consider have a value for this state rather than keeping it blank? It seems odd to have 6 workflow states but only 5 can be accessed through the enum. Easy enough to add a default value on our field which would backfill data.